build/package/circleci/orb.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 description: "Job for finding and sending feature flag code references to LaunchDarkly. Code references documentation: https://docs.launchdarkly.com/v2.0/docs/git-code-references"
 display:
   home_url: https://docs.launchdarkly.com
-  source_url: https://github.com/launchdarkly/ld-find-code-refs/tree/master/build/package/circleci
+  source_url: https://github.com/launchdarkly/ld-find-code-refs/tree/main/build/package/circleci
```

Comment:
ugh - just realized i never update this before the release :(

Reply:
nevermind! looks like redirects are setup 🙏 https://circleci.com/developer/orbs/orb/launchdarkly/ld-find-code-refs

Reply:
woohoo for redirects!